src/dialogs/ImageOverlay.cpp

```diff
@@ -43,7 +43,6 @@ ImageOverlay::ImageOverlay(QPixmap image, QWidget *parent)
 
         screen_ = QGuiApplication::primaryScreen()->availableGeometry();
```

Comment:
This would still make the widget the size of the primary screen, I think my changes fix that, but it would be awesome, if you could verify, that they also work for you! :D

Reply:
works for me! thanks :)